solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpClusterStateProvider.java

```diff
@@ -140,11 +139,11 @@ private ClusterState fetchClusterState(SolrClient client)
       liveNodesTimestamp = System.nanoTime();
     }
 
-    var collectionsNl = (NamedList<Map<String, Object>>) cluster.get("collections");
+    var collectionsNl = (Map<String, Map<String, Object>>) cluster.get("collections");
```

Comment:
this var name is no longer appropriate.  Should be collectionsMap (and we assume by name).